yt_dlp/extractor/nebula.py

```diff
@@ -236,8 +237,8 @@ def _real_extract(self, url):
 
 
 class NebulaClassIE(NebulaBaseIE):
-    IE_NAME = 'nebula:class'
-    _VALID_URL = rf'{_BASE_URL_RE}/(?P<id>[-\w]+)/(?P<ep>\d+)'
+    IE_NAME = 'nebula:class/podcast'
```

Comment:
the `/` in the `IE_NAME` will make for some strange-looking commit msgs, but it doesn't actually mess up the changelog, so I guess it's ok